pjsip/src/pjsip/sip_transport.c

```diff
@@ -1653,6 +1653,11 @@ PJ_DEF(pj_ssize_t) pjsip_tpmgr_receive_packet( pjsip_tpmgr *mgr,
 		    mgr->on_rx_msg(mgr->endpt, PJSIP_ERXOVERFLOW, rdata);
 		    /* Exhaust all data. */
 		    return rdata->pkt_info.len;
+     		} else if (msg_status == PJSIP_EMISSINGHDR) {
+        	    /* No content-length header which is not allowed. */
```

Comment:
This seems to be "some required header was missing", presumably if you send an INVITE with no `to:` header you'll pass through this code too?  Maybe just requires a more conservative log message?